setup.py

```diff
@@ -5,10 +5,11 @@
 
 setuptools.setup(
     name="anomalydetection",
-    version='0.0.1',
-    install_requires=["pandas>=1.2.0"],
+    version='0.0.2',
+    install_requires=["pandas>=1.2.0", "numba"],
```

Comment:
Should we restrict acceptable versions of numba?

Reply:
I see that the restrictions ensure that we all use and test the same versions. However, the dependency resolver and users have more freedom when installing making it more "installation robust". Or what do you think?

Reply:
As long as we are sure that the numba version that is installed provides all the functionality we expect, I am fine with it. I don't know numba well enough to say whether there is some minimum version that we need for our usage.